### ISSUE TITLE
[windows] Tesseract installed languages is not shown in the UI

### ISSUE THREAD

**Issue by bamthomas** [ICIJ/datashare#1009]
**Describe the bug**

On windows with tesseract installer we can choose the OCR languages we want to use. Then when we parse documents with datashare, we can select a preferred language for OCR in a list. But the installed languages are indicated as not installed.

**To Reproduce**

1. Install datashare and tesseract with an additional language (than English)
2. go to task > add documents
3. select the language in the "in which language are your documents?" selector
4. See "OCR for <language> is not installed" (cf screenshot below)

**Expected behavior**

When the language is installed it should be available in datashare.

**Screenshots**

![image](https://user-images.githubusercontent.com/551723/216099425-4ec65792-7c99-4cab-b6fd-a291e93d3490.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: N/A
 - Version: 11.0.3

**Comment by bamthomas** [comment]
it seems that tika is not able to find the tesseract executable :
```
java.io.IOException: Cannot run program "tesseract.exe": CreateProcess error=2, The system cannot find the file specified
        at java.base/java.lang.ProcessBuilder.start(Unknown Source)
        at java.base/java.lang.ProcessBuilder.start(Unknown Source)
        at org.apache.tika.parser.ocr.TesseractOCRParser.preloadLangs(TesseractOCRParser.java:779)
        at org.apache.tika.parser.ocr.TesseractOCRParser.initialize(TesseractOCRParser.java:532)
        at org.icij.datashare.web.SettingsResource.ocrLanguages(SettingsResource.java:90)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
```

**Comment by mvanzalu** [comment]
> it seems that tika is not able to find the tesseract executable :
> 
> ```
> java.io.IOException: Cannot run program "tesseract.exe": CreateProcess error=2, The system cannot find the file specified
>         at java.base/java.lang.ProcessBuilder.start(Unknown Source)
>         at java.base/java.lang.ProcessBuilder.start(Unknown Source)
>         at org.apache.tika.parser.ocr.TesseractOCRParser.preloadLangs(TesseractOCRParser.java:779)
>         at org.apache.tika.parser.ocr.TesseractOCRParser.initialize(TesseractOCRParser.java:532)
>         at org.icij.datashare.web.SettingsResource.ocrLanguages(SettingsResource.java:90)
>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
> ```

Tesseract has to be added to Windows PATH (environment variables)

**Comment by liamgpuk** [comment]
Can confirm adding the Tesseract folder (not the executable) to PATH resolves the issue in Datashare V11.0.3

**Comment by mvanzalu** [comment]
@liamgpuk We fixed it in 11.0.7, the installer adds the path.
